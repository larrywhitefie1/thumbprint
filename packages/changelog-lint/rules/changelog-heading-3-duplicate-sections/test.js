const test = require('../../test-utils');
const rule = require('./');

const valid = [
    '### Foo',
    `### Foo

### Bar

### Baz`,
];

const invalid = [
    `## 1.2.3 - 2010-02-02

### Foo

### Bar

### Foo`,
];

test({
    rule,
    valid,
    invalid,
});
