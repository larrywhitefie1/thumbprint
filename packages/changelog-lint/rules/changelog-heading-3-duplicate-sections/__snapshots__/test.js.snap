// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`invalid case #0 fails 1`] = `
Object {
  "input": "## 1.2.3 - 2010-02-02

### Foo

### Bar

### Foo",
  "output": Array [
    Object {
      "column": 1,
      "fatal": false,
      "line": 7,
      "location": Object {
        "end": Object {
          "column": 8,
          "line": 7,
          "offset": 48,
        },
        "indent": Array [],
        "start": Object {
          "column": 1,
          "line": 7,
          "offset": 41,
        },
      },
      "reason": "Third-level heading “Foo” appears twice within “1.2.3 - 2010-02-02”.",
      "ruleId": "changelog-heading-3-duplicate-sections",
      "source": "remark-lint",
      "stack": null,
    },
  ],
}
`;
